### PR TITLE
Replace String.includes with String.indexOf: cross-browsing support

### PR DESCRIPTION
**Pull request by renatorib**
Usage of `Array.includes` force include polyfills to browser compatibility. Can be easily replaced by indexOf. (I know includes is better, but a full polyfill is too much for only 1 line).

Image below is a project that I are working. Caused fatal error in IE11.
image

### CODE REVIEW

File: src/client/preview/index.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ import reducer from './reducer';
 
 // check whether we're running on node/browser
 const { navigator } = global;
-const isBrowser = navigator && navigator.userAgent !== 'storyshots' && !navigator.userAgent.includes('Node.js');
+const isBrowser = navigator && navigator.userAgent !== 'storyshots' && !navigator.userAgent.indexOf('Node.js') > -1;
```
Comment (UsulPro) [inline src/client/preview/index.js]: It should be in parentheses since `!` has higher precedence than `>`

**Review by usulpro — DISMISSED** [review]

**Comment by ndelangen** [comment]
@renatorib Do you want to finish this PR?, if you could process @UsulPro's review comments, We'll merge this!

**Comment by renatorib** [comment]
@ndelangen fixed

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
Thanks @renatorib !

**Pull request merged**